pkgs/applications/misc/buku/default.nix

```diff
@@ -31,7 +32,7 @@ with pythonPackages; buildPythonApplication rec {
     homepage = https://github.com/jarun/Buku;
     license = licenses.gpl3;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ matthiasbeyer ];
+    maintainers = with maintainers; [ matthiasbeyer infinisil ];
```

Comment:
Well I'd like this package to stay up-to-date, so I guess that's okay 😉. I just got up and running with my NixOS installation and I also intend to contribute much more in the future :D